### PR TITLE
Build bundles with ENB

### LINKED ISSUE #527: Build bundles with ENB

### PR DESCRIPTION
**Pull request by blond**
Resolved #527 

Run:

```
$ npm run build
```

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -20,6 +20,7 @@
     "bem-environ": "~1.2.0"
   },
   "devDependencies": {
+    "vow": "0.4.3",
```
Comment (narqo) [inline package.json]: I haven't noticed any place where you use it. Do we need it here?

File: package.json [hunk 2]
```diff
@@ -27,14 +28,16 @@
...
     "istanbul": "~0.2.6",
-    "enb-bem-docs": "git://github.com/andrewblond/enb-bem-docs.git#6b1544a8f3e516eaee3303cf84131e773868eb2b",
+    "enb-bem-docs": "git://github.com/andrewblond/enb-bem-docs.git#3cd156684d92af632d9cf6337b023fa2c29c0587",
     "enb": "0.10.0",
     "enb-diverse-js": "0.1.0",
     "enb-modules": "0.1.1",
-    "enb-bemxjst": "1.1.1"
+    "enb-bemxjst": "1.1.1",
+    "enb-bh": "0.1.1",
+    "enb-roole": "0.2.0"
```
Comment (narqo) [inline package.json]: Guess this one is unnecessary here?

File: .enb/techs/html-from-bemtree.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
+/**
+ * html-from-bemtree
+ * =================
+ *
+ * Собирает *html*-файл с помощью *bemtree* и *bemhtml*.
```
Comment (narqo) [inline .enb/techs/html-from-bemtree.js:5]: Should we translate this?

**Comment by narqo** [comment]
To be fair, there are a lot of ENB-related configs which I don't understand well.

File: .enb/techs/html-from-bemtree.js (hunk 3 above)
Comment (blond) [inline .enb/techs/html-from-bemtree.js:5]: ENB don't contain documentation in English.

**Comment by blond** [comment]
> To be fair, there are a lot of ENB-related configs which I don't understand well.

It is still better than nothing :) Thanks!

Fixed.

**Pull request merged**